modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Question.es.js

```diff
@@ -13,10 +13,13 @@
  */
 
 import ClayAlert from '@clayui/alert';
-import ClayButton from '@clayui/button';
+import ClayButton, {ClayButtonWithIcon} from '@clayui/button';
```

Comment:
About this file, I think we should duplicate this file, create the `Question.new.es.js` and use this new component in UserActivity.new.js, because both files will run under feature flag, and touching this file may lead other problems

Can you duplicate the file as I said, and undo the changes to this one?